src/routes/home/index.tsx

```diff
@@ -1,5 +1,5 @@
 import { h, Component } from 'preact';
-import Card from 'preact-material-components/Card';
+import Card, { Primary, Title, Subtitle, SupportingText, Actions, Action } from 'preact-material-components/Card/Card';
```

Comment:
Obviously not ideal to have to import like this, but also it does not work. TypeScript does not complain since the typings are technically correct, but webpack errors with "export was not found" (as the actual js code does not have these exported separately).